wiki/Layout-component/Lead-space-search.md

```diff
@@ -15,7 +15,6 @@ The [[Lead space search]] is in the same family of our other Lead space componen
 - [[Search typeahead]]
 - [[Tag group]]
 - [[Carbon breadcrumb]]
-- [[Lead space wrapper]]
```

Comment:
Wouldn't Lead space search need `Background media` too?

Reply:
I ended up removing it here because it doesn't depend on it yet.

Reply:
Ah yes make sense! TY